code/modules/projectiles/gun.dm

```diff
@@ -76,7 +76,8 @@
 	build_zooming()
 
 
-/obj/item/weapon/gun/CheckParts()
+/obj/item/weapon/gun/CheckParts(list/contents)
```

Comment:
typo in the argument, "content" not "contents". This wouldn't happen if the arg name was not so similar to a var name.